### PR TITLE
[Cape Town 2020] Make welcome page more welcoming

### PR DESCRIPTION
**Pull request by adrianmoisey**

### CODE REVIEW

**Comment by adrianmoisey** [comment]
Waiting on review from another team member before merging.

File: content/events/2020-cape-town/welcome.md [hunk 1]
```diff
@@ -1,87 +1,102 @@
...
+            <i class="fa fa-book fa-lg"></i>&nbsp;&nbsp;&nbsp;Submit a talk
+          </a>
+        </div>
+        </div>
+      </div>
+    </div><!-- end a content element -->
+    <div class = "row">
+      <div class = "col-12">
+        <div class="text-center">
+        <img src = "/events/2019-cape-town/logo.png" class = "img-fluid" style="width: 50%">
```
Comment (bridgetkromhout) [inline content/events/2020-cape-town/welcome.md]: Please don't reference a past year - copy the content to the current year static assets path.

Comment (adrianmoisey) [reply content/events/2020-cape-town/welcome.md]: Sorry! Was copy/paste error

Comment (adrianmoisey) [reply content/events/2020-cape-town/welcome.md]: Fixed in https://github.com/devopsdays/devopsdays-web/pull/9008/commits/03932dabf4f8e2cb4d4642dbed247553dd901759

**Comment by manicminer** [comment]
LGTM 👍

**Pull request merged**